src/main/kotlin/com/cjbooms/fabrikt/generators/controller/metadata/Spring.kt

```diff
@@ -41,6 +43,8 @@ object SpringImports {
 
     val AUTHENTICATION = ClassName(Packages.SPRING_AUTHENTICATION, "Authentication")
 
+    val COMPLETABLE_FUTURE = ClassName(Packages.JAVA_UTIL_CONCURRENT, "CompletableFuture")
```

Comment:
Would the interface be better? eg CompletionStage

Reply:
Probably cleaner, although there is nothing else in the jdk the extends the interface.

Reply:
changed